libs/components/indicators/src/lib/modules/wait/wait.component.spec.ts

```diff
@@ -546,6 +553,172 @@ describe('Wait component', () => {
       expect(getAriaLabel()).toBe('Page loading.');
     });
 
+    it('should announce the ariaLabel when loading begins and the screenReaderCompletedText when it ends to screen readers', () => {
+      const fixture = TestBed.createComponent(SkyWaitTestComponent);
```

Comment:
There are large blocks of copy/pasted code in these tests. Factor these into one or more functions that can be called with parameters for the values and/or behaviors that differ between tests.

Reply:
Done